docs/IE_DG/supported_plugins/GNA.md

```diff
@@ -159,6 +162,30 @@ Heterogeneous plugin was tested with the Intel&reg; GNA as a primary device and
 
 > **NOTE:** Due to limitation of the Intel&reg; GNA backend library, heterogenous support is limited to cases where in the resulted sliced graph, only one subgraph is scheduled to run on GNA\_HW or GNA\_SW devices.
 
+## Recovery from interruption by high-priority Windows audio processes\*
+
+As noted in the introduction, GNA is designed for real-time workloads such as noise reduction.
+For such workloads, processing should be time constrained, otherwise extra delays may cause undesired effects such as
+audio "glitches". To make sure that processing can satisfy real time requirements, the GNA driver provides a QoS
+(Quality of Service) mechanism which interrupts requests that might cause high-priority Windows audio processes to miss
+schedule, thereby causing long running GNA tasks to terminate early.
+
+Applications should be prepared for this situation.
+If an inference (in `GNA_HW` mode) cannot be executed because of such interruption, then `InferRequest::Wait()` will return status code
+`StatusCode::INFER_NOT_STARTED` (note that it will be changed to a more meaningful status code in future releases).
+
+Any application working with GNA must properly react if receives this code. Various strategies are possible.
+One of the options is to immediately switch to GNA SW emulation mode:
+
+```cpp
+std::map<std::string, Parameter> newConfig;
+newConfig[GNAConfigParams::KEY_GNA_DEVICE_MODE] = Parameter("GNA_SW_EXACT");
+executableNet.SetConfig(newConfig);
+
+```
+
+and switch back to GNA_HW after some time hoping that the competing application has finished.
```

Comment:
resubmit the infer request, and switch back

Reply:
Good catch! Fixed